main.py

```diff
@@ -33,8 +34,15 @@ def create_menu():
 
 def add_book():
     new_book = ui.get_book_info()
-    new_book.save()
-    
+    try:
+        if store.exact_match(new_book):
+            print('Looks like you already have this book in your book list. Try again')
+        else:
+            new_book.save()
+            print('Added new book!')
+    except ValueError as err:
```

Comment:
How does ValueError get thrown here? Address why this error is being raised - you can prevent it in code and remove the try-except here. Please re-open this issue and fix.